zou/app/services/shots_service.py

```diff
@@ -266,6 +266,81 @@ def get_shots_and_tasks(criterions={}):
     return list(shot_map.values())
 
 
+def get_entities_and_tasks(criterions={}):
```

Comment:
Can you move this in the entities_service?

Reply:
Done